bin/semantic-release.js

```diff
@@ -23,8 +23,6 @@ npx is bundled with npm >= 5.4, or available via npm. More info: npm.im/npx`
 }
 
 // node 8+ from this point on
-require('../src')().catch(err => {
-  console.error('An error occurred while running semantic-release');
-  console.error(err);
-  process.exit(1);
+require('../src/cli')().catch(() => {
```

Comment:
Why is this error not logged anymore?

Reply:
It is logged here: https://github.com/semantic-release/semantic-release/blob/caribou/src/cli.js#L56

In addition it's tested there: https://github.com/semantic-release/semantic-release/blob/caribou/test/integration.test.js#L618